include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -275,10 +295,30 @@ bool setupReducers() { return detail::tl_status.setup_reducers; }
 RAJA_INLINE
 cuda_dim_t currentGridDim() { return detail::tl_status.gridDim; }
 
+//! get grid size of current launch
+RAJA_INLINE
+cuda_dim_member_t currentGridSize() { return detail::tl_status.gridDim.x *
+                                             detail::tl_status.gridDim.y *
+                                             detail::tl_status.gridDim.z; }
+
 //! get blockDim of current launch
 RAJA_INLINE
 cuda_dim_t currentBlockDim() { return detail::tl_status.blockDim; }
 
+//! get block size of current launch
+RAJA_INLINE
+cuda_dim_member_t currentBlockSize() { return detail::tl_status.blockDim.x *
+                                              detail::tl_status.blockDim.y *
+                                              detail::tl_status.blockDim.z; }
+
+//! get dynamic shared memory for current launch, you may add to this
+RAJA_INLINE
+size_t& currentDynamicSmem() { return *detail::tl_status.dynamic_smem; }
```

Comment:
```suggestion
size_t& currentDynamicShmem() { return *detail::tl_status.dynamic_smem; }
```
Is it a better name with the 'h' included? I think so. If you agree to change it, please do so here and in all the method and variable names below.

Reply:
done